web_src/js/utils/match.ts

```diff
@@ -41,3 +43,114 @@ export function matchMention(queryText: string): string[] {
 
   return sortAndReduce(results);
 }
+
+type Issue = {state: 'open' | 'closed'; pull_request: {draft: boolean; merged: boolean} | null};
+type IssueMention = {value: string; name: string; issue: Issue};
+export async function matchIssue(url: string, queryText: string): Promise<IssueMention[]> {
+  const query = queryText.toLowerCase();
+
+  // http://localhost:3000/anbraten/test/issues/1
+  // http://localhost:3000/anbraten/test/compare/main...anbraten-patch-1
+  const repository = (new URL(url)).pathname.split('/').slice(1, 3).join('/');
```

Comment:
It could be `/sub-path/owner/repo/issues/1`, see `parseIssueHref` and its tests

Reply:
awesome, exactly what I needed.